kiota_abstractions/utils.py

```diff
@@ -0,0 +1,26 @@
+import importlib.util
+import sys
+
+
+def lazy_import(name):
+    """Lazily imports a python module given its absolute path
+
+    Args:
+        name (str): Absolute path to the module
+
+    Returns:
+        module: The module to be imported
+    """
+    spec = importlib.util.find_spec(name)
```

Comment:
there should probably be some defensive programing in place

Reply:
Thanks. Done in https://github.com/microsoft/kiota-abstractions-python/pull/19/commits/aa4c29726a09d763c66181f694dd13c3bb7be701